src/box/sql/resolve.c

```diff
@@ -202,13 +218,30 @@ sql_find_column_expr(const struct ExprList *list, const char *tab_name,
  * in pParse and return WRC_Abort.  Return WRC_Prune on success.
  */
 static int
-lookupName(Parse * pParse,	/* The parsing context */
-	   const char *zTab,	/* Name of table containing column, or NULL */
-	   const char *zCol,	/* Name of the column. */
-	   NameContext * pNC,	/* The name context used to resolve the name */
-	   Expr * pExpr		/* Make this EXPR node point to the selected column */
-    )
+lookupName(struct Parse *pParse, struct Expr *pExpr, struct NameContext *pNC)
 {
+	const char *zTab = NULL;
+	const char *zCol = NULL;
+	char *old_tab = NULL;
+	char *old_col = NULL;
+	const struct Expr *col = NULL;
+	if (pExpr->op == TK_DOT) {
+		assert(pExpr->pLeft->op == TK_ID && pExpr->pRight->op == TK_ID);
```

Comment:
Why db.table.col is not handled here?

Reply:
Now we only have one DB. So there is no cases `db.table.field`.

Reply:
Alright, then everything is fine here. Thanks for clarification!